### PR TITLE
Testing/Clint Kunz

### PR DESCRIPTION
**Pull request by clintKunz**
@bradiowave

### CODE REVIEW

File: basic-javascript/project-1.spec.js [hunk 1]
```diff
@@ -1,3 +1,245 @@
...
+    it('should return 0 when called with no value', () => {
+        const product = helpers.multiplyByTen();
+        expect(product).toBe(0);
+    });
+
+    // //string input
+    it('should throw error when called with a string', () => {
+        expect(() => {
+            helpers.multiplyByTen('five');
+        }).toThrow();
```
Comment (bradencodes) [inline basic-javascript/project-1.spec.js:22]: Good test to make sure input data is valid.

File: basic-javascript/project-1.spec.js [hunk 2]
```diff
@@ -1,3 +1,245 @@
...
+        expect(answer).toEqual(expected);
+    });
+});
+
+//less than 90
+describe('lessThanNinety()', () => {
+    it('should say if it is less than 90', () => {
+        const expected = true;
+        const answer = helpers.lessThanNinety(75);
+        expect(answer).toEqual(expected);
```
Comment (bradencodes) [inline basic-javascript/project-1.spec.js:60]: Also include a test where it expects false when the value is greater than or equal to 90.

File: package.json [hunk 3]
```diff
@@ -4,12 +4,21 @@
   "description": "Practice exercises to learn the syntax for testing javascript code.",
   "main": "index.js",
   "scripts": {
-    "test": "jest --watch"
+    "test": "jest --watch --verbose"
```
Comment (bradencodes) [inline package.json:7]: It's always good to make your tests verbose so you can see which ones passed and which ones failed 👍

**Comment by bradencodes** [comment]
## MVP
- [x] Write tests for project-1.js
- [x] Write tests for project-2.js

## Stretch
- [ ] Correct bugs and refactor the code
- [ ] Write tests for arrays.js
- [ ] Write tests for object.js

## What Looks Great
- MVP complete! Nice work
- Code is very nicely formatted and easy to read. Every test has a `describe` and several `it`s.
- Titles for tests are descriptive

## What Could Be Improved
- Need more commits. You should have one commit for every one change.
- Commits need more description. Check out this guide (https://chris.beams.io/posts/git-commit/)
- Tests need to be more thorough and check for edge cases. Also check for both expected success and expected fail cases for each function.

## What Could Be Added
- Refactor some of the code to make it pass the test (stretch)

## Rating (1-3)
**2**
